internal/server/gpgEndpoints.go

```diff
@@ -75,10 +85,20 @@ func (ge *GPGEndpoint) decrypt(w http.ResponseWriter, r *http.Request) {
 
 	w.Header().Set("Content-Type", models.MimeJSON)
 	w.WriteHeader(200)
-	n, _ := w.Write([]byte(d))
+	n, _ := w.Write(d)
 	LogExit(log, r, 200, n)
 }
 
+// Encrypt godoc
+// @id gpg-data-encrypt
+// @tags GPG Operations
+// @Summary Encrypts data for a GPG Key
```

Comment:
A `GPG Key` mencionada aqui faz parte dos params da função?
(desculpe que não sei ler Go)

Pergunto pois acima falamos da `previously loaded GPG Private Key`. E acho que seria melhor contextualizar melhor.

Reply:
Faz sim! No `encrypt` passa o fingerprint como argumento assim como na `decrypt`. Mas acho que talvez vale especificar igual no `decrypt`: 
`Encrypts data for a previously loaded GPG Public Key` , pois ela precisa estar carregada (é só o fingerprint o argumento)